samples/subsys/usb/dfu/README.rst

```diff
@@ -0,0 +1,83 @@
+.. _usb_dfu:
+
+USB DFU Sample Application
+##########################
+
+Overview
+********
+
+This sample app demonstrates use of a USB DFU Class driver provided
+by the Zephyr project.
+
+Requirements
+************
+
+This project requires an USB device driver. Currently, the USB DFU
+class provided by the Zephyr project depends on DFU image manager and
+partition layout. Refer to :ref:`flash_partitions` for details about
+partition layout. You SoC have to run MCUboot as stage 1 bootloader.
```

Comment:
change to:

    Your SOC must run MCUboot as the stage 1 bootloader.  This sample
    is built as an application for the MCUboot bootloader.